src/bin/pg_autoctl/cli_show.c

```diff
@@ -82,6 +84,7 @@ CommandLine show_events_command =
 				 "  --formation   formation to query, defaults to 'default' \n"
 				 "  --group       group to query formation, defaults to all \n"
 				 "  --count       how many events to fetch, defaults to 10 \n"
+				 "  --watch       display an auto-updating dashboard\n"
```

Comment:
Should this still show the state too? Or only the events. Right now it displays both.

Reply:
Yeah we have a single watch command and 3 ways to call it. I think I like that we can add `--watch` to the existing commands and have the new dashboard, even though it's not just `show state` or `show events` anymore. I would vote for keeping it that way?